src/Stormpath/Authc/UsernamePasswordRequest.php

```diff
@@ -22,12 +22,15 @@ class UsernamePasswordRequest implements AuthenticationRequest
     private $username;
     private $password;
     private $host;
+    private $accountStore;
 
-    public function __construct($username, $password, $host = null)
+    public function __construct($username, $password, array $options = array())
```

Comment:
I honestly don't think it matters. I don't think that customers are using this so this is probably a better approach for this. @omgitstom what do you think?